Products/ZenUtils/zenpack.py

```diff
@@ -755,12 +757,12 @@ def buildOptions(self):
                                action="store_true",
                                default=False,
                                help='List installed ZenPacks')
-        self.parser.add_option('--keep-packs',
+        self.parser.add_option('--keep-pack',
                                dest='keepPackNames',
```

Comment:
Because the option can be specified multiple times and will generate a pack name list so it should be ok here.